docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,127 @@
+					- change the system type to: [secondary_system_type[0], "G3_1_1b"] - remember that expected_system_type_from_Table_G3_1_1 gives both the expected system type and a string explaining how that system type was chosen.  For the rule testing G3_1_1b, it is only asking whether the system was chosen from Table G3.1.1, not exactly how it got there: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = secondary_system_type[0]`
+					- change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1b"`
+
+- G3.1.1c "If the baseline HVAC system type is 5, 6, 7 or 8 use separate single-zone systems conforming with the requirements of system 3 or system 4 for any HVAC zones that have occupancy, internal gains, or schedules that differ significantly from the rest of the HVAC zones served by the system. The total peak internal gains that differ by 10 Btu/h·ft2 or more from the average of other HVAC zones served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+- create a list of zones that meet G3.1.1c - we can't change the system type assignment as we go, otherwise we'll be calculating the peak load average with a diminishing list of zones: `zones_that_meetG3_1_1c_list = []`
+- loop through the zones_and_systems to see if any of the zones meets G3.1.1.c: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1c to determine if this zone meets this requirement: `if does_zone_meet_G3_1_1c(B_RMI,zone,zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] == YES):`
+		- append the zone_id to the zones_that_meetG3_1_1c_list: `zones_that_meetG3_1_1c_list.append(zone)`
+- loop through zones_that_meetG3_1_1c_list and change the expected system type and origin for the zones that meet G3.1.1c: `for zone in zones_that_meetG3_1_1c_list:`
+	- change the system origin string: `zones_and_systems[zone]["SYSTEM_ORIGIN"] = "G3_1_1c"` 
+	- the zone meets the G3_1_1c requirements, choose system 3 or 4 based on climate zone: `if is_CZ_0_to_3a():`
+		- set the system to SYS_4: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_4`
+	- else (the climate zone is_CZ_3b_3c_and_4_to_8): `else:`
+		- set the system to SYS_3: `zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_3`
+				
+- G3.1.1d "For laboratory spaces in a building having a total laboratory exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces.  The lab exhaust fan shall be modeled as constant horsepower reflecting constantvolume stack discharge with outdoor air bypass."
+	- not sure how to apply this one
```

Comment:
Should this be evaluated for applicability only?

Reply:
LABORATORY is one of the SpaceFunctionOptions.  I've taken a crack at creating a function - [does_zone_meet_G3_1_1d](https://github.com/pnnl/ruleset-checking-tool/pull/1095)

If we are evaluating for applicability only, I think it makes sense to do this at the rule level - [Rule 18-26](https://onedrive.live.com/edit.aspx?resid=796769604E905084!462&cid=796769604e905084&authkey=!Ap-zMmcPgQlWt0Q&CT=1683226254245&OR=ItemsView)